aspnetcore/host-and-deploy/windows-service.md

```diff
@@ -2,59 +2,106 @@
 title: Host ASP.NET Core in a Windows Service
 author: guardrex
 description: Learn how to host an ASP.NET Core app in a Windows Service.
-monikerRange: '>= aspnetcore-2.1'
+monikerRange: '>= aspnetcore-2.2'
 ms.author: tdykstra
 ms.custom: mvc
-ms.date: 10/30/2018
+ms.date: 11/26/2018
 uid: host-and-deploy/windows-service
 ---
 # Host ASP.NET Core in a Windows Service
 
 By [Luke Latham](https://github.com/guardrex) and [Tom Dykstra](https://github.com/tdykstra)
 
-An ASP.NET Core app can be hosted on Windows without using IIS as a [Windows Service](/dotnet/framework/windows-services/introduction-to-windows-service-applications). When hosted as a Windows Service, the app automatically starts after reboots.
+An ASP.NET Core app can be hosted on Windows as a [Windows Service](/dotnet/framework/windows-services/introduction-to-windows-service-applications) without using IIS. When hosted as a Windows Service, the app automatically starts after reboots.
 
 [View or download sample code](https://github.com/aspnet/Docs/tree/master/aspnetcore/host-and-deploy/windows-service/samples) ([how to download](xref:index#how-to-download-a-sample))
 
+## Deployment type
+
+You can create either of the following Windows Service deployments:
+
+* Framework-dependent deployment (FDD) &ndash; FDD relies on the presence of a shared system-wide version of .NET Core on the target system. When the FDD scenario is used with an ASP.NET Core Windows Service app, the SDK produces an executable (*\*.exe*), called a *framework-dependent executable*.
+
+* Self-contained deployment (SCD) &ndash; SCD doesn't rely on the presence of shared components on the target system. The runtime and the app's dependencies are deployed with the app to the hosting system.
+
+For more information on the deployment scenarios and advice on which one to choose, see [.NET Core application deployment](/dotnet/core/deploying/).
+
 ## Convert a project into a Windows Service
 
-The following minimum changes are required to set up an existing ASP.NET Core project to run as a service:
+Make the following changes to an existing ASP.NET Core project to run the app as a service:
 
 1. In the project file:
 
-   * Confirm the presence of a Windows [Runtime Identifier (RID)](/dotnet/core/rid-catalog) or add it to the `<PropertyGroup>` that contains the target framework:
+   * **Framework-dependent Deployment (FDD)** &ndash; Add a Windows [Runtime Identifier (RID)](/dotnet/core/rid-catalog) to the `<PropertyGroup>` that contains the target framework. Add the `<SelfContained>` property set to `false`. Disable the creation of a *web.config* file by adding the `<IsTransformWebConfigDisabled>` property set to `true`.
```

Comment:
Make it more obvious that the reader is to choose between FDD or SCD at this point.